tests/test_bookkeeper.py

```diff
@@ -336,8 +475,6 @@ def test_bookkeeping_onchaind_txs(node_factory, bitcoind):
 
     # Double check there's no bookkeeper plugin on
     assert l1.daemon.opts['disable-plugin'] == 'bookkeeper'
-    with pytest.raises(RpcError):
-        l1.rpc.listincome()
```

Comment:
Why remove this??

Reply:
It fails now b/c the command name moved to `bkpr_listincome`, not because it's not there. It's technically a functional check of the above (deleting `bookkeeper` plugin)